engine/src/main/scala/cromwell/webservice/WorkflowSubmitBatchActor.scala

```diff
@@ -34,7 +35,9 @@ case class WorkflowManagerBatchIndexedResponse(response: WorkflowManagerResponse
 class WorkflowSubmitIndexedActor(batchSubmitActor: ActorRef, requestHandlerActor: ActorRef,
```

Comment:
IMHO that's a fine suggestion for an enhancement ticket, but beyond the scope of this "make it great again" PR.  😄